src/infra/src/table/short_urls.rs

```diff
@@ -258,8 +238,7 @@ pub async fn batch_remove(short_ids: Vec<String>) -> Result<(), errors::Error> {
     Entity::delete_many()
         .filter(Column::ShortId.is_in(short_ids))
         .exec(client)
-        .await
-        .map_err(|e| Error::DbError(DbError::SeaORMError(e.to_string())))?;
+        .await?;
```

Comment:
This logic moved to [this trait implementation](https://github.com/openobserve/openobserve/pull/5100/files#r1847373471) so it should now happen automatically when `?` converts the error.